cvat-core/src/lambda-manager.ts

```diff
@@ -14,7 +14,12 @@ export interface ModelProvider {
     attributes: Record<string, string>;
 }
 
-interface ModelProxy {
+export enum ModelRequestStatus {
+    QUEUED = 'queued',
+    STARTED = 'started',
+}
```

Comment:
We should find a way to remove all the code related with models from the core application